core/src/main/java/com/datastax/oss/driver/internal/core/util/ArrayUtils.java

```diff
@@ -77,7 +78,7 @@ public static <ElementT> void shuffleHead(@NonNull ElementT[] elements, int n) {
    *     Fisher-Yates shuffle</a>
    */
   public static <ElementT> void shuffleHead(
-      @NonNull ElementT[] elements, int n, @NonNull ThreadLocalRandom random) {
+      @NonNull ElementT[] elements, int n, @NonNull Random random) {
```

Comment:
[Mockito says](https://github.com/mockito/mockito/issues/2985) they can no longer support mocking of `ThreadLocalRandom`.
This fix will also close [JAVA-3137](https://datastax-oss.atlassian.net/browse/JAVA-3137).